### ISSUE TITLE
build-check problem with R 3.6.0

### ISSUE THREAD

**Issue by dankelley** [dankelley/oce#1536]
I'm attaching a transcript from OSX with all the details.  (R now produces a lot more info when problems arise, which is a good thing!)

I also have an `rhub::check_on_linux` job going that ought to produce results before noon. I'd like to fix this up during the afternoon. The first error message rings a bell, and will likely be fixed very easily.

[err.txt](https://github.com/dankelley/oce/files/3125053/err.txt)

**Comment by dankelley** [comment]
I think this may be related to https://github.com/mailund/pmatch/issues/65#issuecomment-460972778

**Comment by dankelley** [comment]
I'm getting through these problems one at a time. They all stem from cases in which I write e.g. `if (X)` where `length(X)>1`, or similar problems with `&&` operators.  The build-test procedure is very slow, taking maybe 10 to 15 minutes to uncover a problem, but I just realized that if I first do
```R
Sys.setenv("_R_CHECK_LENGTH_1_CONDITION_"=TRUE)
Sys.setenv("_R_CHECK_LENGTH_1_LOGIC2_"=TRUE)
```
in an interactive session, then I can get these problems to arise with a simple `devtools::test()`, as opposed to the more extensive `devtools::check()` (or the menu items that lead to these, or something like these).

**Comment by dankelley** [comment]
Fixed in "develop" branch, commit 491574cb871b105d5807eac93e1c78715cff03b5

1. Build/test/check OK within RStudio.
2. In the commandline, the following also works
```
R CMD BUILD oce
R CMD CHECK --as-cran oce_1.0-2.1.tar.gz
R CMD INSTALL oce_1.0-2.1.tar.gz
```
